fhir-model/src/main/java/com/ibm/fhir/model/lang/util/LanguageRegistry.java

```diff
@@ -0,0 +1,9012 @@
+        l.add("gud");
+        l.add("gue");
+        l.add("guf");
+        l.add("gug");
+        l.add("guh");
+        l.add("gui");
+        l.add("guk");
+        l.add("gul");
+        l.add("gum");
+        l.add("gun");
+        l.add("guo");
+        l.add("gup");
+        l.add("guq");
+        l.add("gur");
+        l.add("gus");
+        l.add("gut");
+        l.add("guu");
+        l.add("guw");
+        l.add("gux");
+        l.add("guz");
+        l.add("gva");
+        l.add("gvc");
+        l.add("gve");
+        l.add("gvf");
+        l.add("gvj");
+        l.add("gvl");
+        l.add("gvm");
+        l.add("gvn");
+        l.add("gvo");
+        l.add("gvp");
+        l.add("gvr");
+        l.add("gvs");
+        l.add("gvy");
+        l.add("gwa");
+        l.add("gwb");
+        l.add("gwc");
+        l.add("gwd");
+        l.add("gwe");
+        l.add("gwf");
+        l.add("gwg");
+        l.add("gwi");
+        l.add("gwj");
+        l.add("gwm");
+        l.add("gwn");
+        l.add("gwr");
+        l.add("gwt");
+        l.add("gwu");
+        l.add("gww");
+        l.add("gwx");
+        l.add("gxx");
+        l.add("gya");
+        l.add("gyb");
+        l.add("gyd");
+        l.add("gye");
+        l.add("gyf");
+        l.add("gyg");
+        l.add("gyi");
+        l.add("gyl");
+        l.add("gym");
+        l.add("gyn");
+        l.add("gyo");
+        l.add("gyr");
+        l.add("gyy");
+        l.add("gza");
+        l.add("gzi");
+        l.add("gzn");
+        l.add("haa");
+        l.add("hab");
+        l.add("hac");
+        l.add("had");
+        l.add("hae");
+        l.add("haf");
+        l.add("hag");
+        l.add("hah");
+        l.add("hai");
+        l.add("haj");
+        l.add("hak");
+        l.add("hal");
+        l.add("ham");
+        l.add("han");
+        l.add("hao");
+        l.add("hap");
+        l.add("haq");
+        l.add("har");
+        l.add("has");
+        l.add("hav");
+        l.add("haw");
+        l.add("hax");
+        l.add("hay");
+        l.add("haz");
+        l.add("hba");
+        l.add("hbb");
+        l.add("hbn");
+        l.add("hbo");
+        l.add("hbu");
+        l.add("hca");
+        l.add("hch");
+        l.add("hdn");
+        l.add("hds");
+        l.add("hdy");
+        l.add("hea");
+        l.add("hed");
+        l.add("heg");
+        l.add("heh");
+        l.add("hei");
+        l.add("hem");
+        l.add("hgm");
+        l.add("hgw");
+        l.add("hhi");
+        l.add("hhr");
+        l.add("hhy");
+        l.add("hia");
+        l.add("hib");
+        l.add("hid");
+        l.add("hif");
+        l.add("hig");
+        l.add("hih");
+        l.add("hii");
+        l.add("hij");
+        l.add("hik");
+        l.add("hil");
+        l.add("him");
+        l.add("hio");
+        l.add("hir");
+        l.add("hit");
+        l.add("hiw");
+        l.add("hix");
+        l.add("hji");
+        l.add("hka");
+        l.add("hke");
```

Comment:
read this from a file... please

Reply:
The byte code is going to embed all of this... if you externalize to src/main/resources at least it's not in the core byte code

Reply:
Will do.  Thanks!